easybuild/easyconfigs/m/methylpy/methylpy-1.2.9-foss-2021b-fixSetupConfig.patch

```diff
@@ -0,0 +1,7 @@
+--- setup.cfg.orig	2022-01-19 19:34:38.630926000 +0100
++++ setup.cfg	2022-01-19 19:34:44.636856000 +0100
+@@ -1,2 +1,2 @@
+ [metadata]
+-description-file = README.md
++description_file = README.md
```

Comment:
For a simple change like this, why not use a `sed` command in `preinstallopts` instead, and remove the patch file?

```python
preinstallopts = "sed -i 's/description-file/description_file/g' setup.cfg && "
```

(the naming of the patch file could also be better, since the change is totally unrelated to the toolchain)